### ISSUE TITLE
Wick Bot user

### ISSUE THREAD

**Issue by Mattie1022** [premint/premintbot#10]
Wick Bot is giving issue to add Premint-config, channel !

**Comment by mager** [comment]
Hey @Mattie1022, did you have trouble adding the bot?

**Comment by Mattie1022** [comment]
Yes, even after i turn it off.. also added Premint bot as whitelisted in wick dashboard. then i try to manually add permission, no luck, Thanks

**Comment by mager** [comment]
You shouldn't need to manually modify permissions.

Can you please share your Discord guild name and/or guild ID?

Try running `!premint-setup` to see the setup status.
